kong/plugins/grpc-gateway/protojson.lua

```diff
@@ -0,0 +1,600 @@
+local protojson = {
```

Comment:
This file should be in `kong/tools/`.
Maybe we should make grpc a folder, and rename `grpc.lua` to `init.lua`, and put both files into the folder.

Reply:
https://github.com/Kong/kong/pull/8297/commits/c4b8f59e86cf5718c47737c0698576fabd5b1fe4

Reply:
> What about the file protobuf.lua? It looks like representation of https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/struct.proto but I am not sure

Looks good to me.

Reply:
It was more about placing all proto-related stuff into one folder. `protobuf.lua` looks like part of {`grpc.lua`, `protojson.lua`} family. Nevertheless, I would personally prefer flat structure, as these files have no mutual dependencies.